### ISSUE TITLE
Documentation hard to follow when instantiating k8srestconfig.Config

### ISSUE THREAD

**Issue by DanielCalvo** [giantswarm/operatorkit#304]
Hey everyone!

I've started looking at operator kit as I've forked giantswarm/net-exporter to write my own exporter, which uses operatorkit to connect to a cluster.

The example encountered in [k8srestconfig#Config](https://godoc.org/github.com/giantswarm/operatorkit/client/k8srestconfig#Config) was sadly not straightforward enough to follow for someone without in-depth knowledge of operator-kit.

Could it be possible to have an example with the actual raw values in the code/doc? Or perhaps a clearer explanation on how to obtain those values for people who are getting started with operator-kit?

For instance, I think kubernetes/client-go has a very good example of an example: [main.go](https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go)

My current use: Connecting to a cluster on GCP from my local network.

**Comment by xh3b4sd** [comment]
Hey, I am so sorry this issue did not get attention. Assuming this is not relevant anymore I just close it. If you still need help with this please reopen. In the future you can ping `@giantswarm/sig-operator` to get attention. ❤️
